service/history/ndc/workflow_resetter_test.go

```diff
@@ -991,9 +991,10 @@ func (s *workflowResetterSuite) TestReapplyContinueAsNewWorkflowEvents_ExcludeAl
 	baseNextEventID := int64(456)
 	baseBranchToken := []byte("some random base branch token")
 	optionExcludeAllReapplyEvents := map[enumspb.ResetReapplyExcludeType]struct{}{
-		enumspb.RESET_REAPPLY_EXCLUDE_TYPE_SIGNAL: {},
-		enumspb.RESET_REAPPLY_EXCLUDE_TYPE_UPDATE: {},
-		enumspb.RESET_REAPPLY_EXCLUDE_TYPE_NEXUS:  {},
+		enumspb.RESET_REAPPLY_EXCLUDE_TYPE_SIGNAL:         {},
+		enumspb.RESET_REAPPLY_EXCLUDE_TYPE_UPDATE:         {},
+		enumspb.RESET_REAPPLY_EXCLUDE_TYPE_NEXUS:          {},
+		enumspb.RESET_REAPPLY_EXCLUDE_TYPE_CANCEL_REQUEST: {},
```

Comment:
No this is needed to get one test passing with latest API changes, because of api-go pipeline being broken, non-versioning changes also crawled into the api-go commit :( (the same line change is in https://github.com/temporalio/temporal/pull/6888)